### PR TITLE
integrated database into the project, fixes done : 'makemigrations an…

### PR DESCRIPTION
**Pull request by ahmedabougabal**
…d migrate crashed due to absence of a database user thus connection failed'

faced some hardships fixing the database and linking it to a user , after some tries, i was able to rerun makemigrations and migrate successfully without any errors encountered!
before : 
image

after : 
image

### CODE REVIEW

**Comment by ahmedabougabal** [comment]
testing and finalizing : 
image
Products :100: 
image

**Comment by ahmedabougabal** [comment]
here is the missing mart screenshot :D 

image

**Comment by ahmedabougabal** [comment]
Authentication and its route is functional now

**Comment by MennaEwas** [comment]
very Nice

File: Products/migrations/0001_initial.py
Comment (MennaEwas) [inline Products/migrations/0001_initial.py:1]: I think migrations should be at the .gitignore level

File: mart/views.py [hunk 1]
```diff
@@ -22,4 +22,22 @@ def welcome(request):
...
+    selected_products = request.GET.getlist('products')
+    products = {
+        "sunglasses": 500,
+        "microwave": 700,
+        "tshirt": 230,
+    }
+    cart_products = []
+    total = 0
+    for product in selected_products:
+        if product in products:
```
Comment (MennaEwas) [inline mart/views.py:37]: Where is the serializer part?

**Review by MennaEwas — APPROVED** [review]
Try to add serializer

File: mart/views.py (hunk 1 above)
Comment (ahmedabougabal) [reply mart/views.py:37]: It has been added, you may review now :)

**Pull request merged**